### DISCUSSION TITLE
"Mac Mouse Fix" doesn't stay enabled

### DISCUSSION THREAD

**Discussion by PuzzleHeaded555** [noah-nuebling/mac-mouse-fix#228] [Questions] [answered]
During the day it will just disable itself. Also on reboot even when I have it set to start on login it doesn't stay enabled. When it's disabled, my scrolling stops working. 

MacBook Pro 2019 - Catilina
Microsoft Trackball Explorereer 1.0 

I do love that my old trackball back/next buttons work again.

**Comment by noah-nuebling** [comment]
Hi @anim8rav, thanks for reaching out about this!

That's quite a strange issue, I've never heard of it. If you could record some console logs or find some crash reports, it would be awesome if you could send them through. Info on how to gather console logs and crash reports is on the [Bug Report page](https://noah-nuebling.github.io/mac-mouse-fix-feedback-assistant/?type=bug-report).

You can either create a _new_ [Bug Report](https://noah-nuebling.github.io/mac-mouse-fix-feedback-assistant/?type=bug-report) or you could post any logs/reports you find in this thread.

Thx!

Oh and by the way: You don't have to add Mac Mouse Fix to login items! It should stay enabled even after you close it. I know it's not super clear. It works this way because MMF used to be a preference pane, but Big Sur made it almost impossible to develop those so I had to switch to an app. I plan to refresh the UI at some point to make it more obvious.

**Comment by noah-nuebling** [comment, marked as answer]
Hey! Marking this as answered to clean things up since I moved the topic to Issue #230.
